model/shared/src/main/scala/queries/schemas/itc/syntax.scala

```diff
@@ -38,8 +39,8 @@ trait syntax:
           .orElse(
             GmosCcdMode(
               if (isFPU) GmosXBinning.One
-              else xbinNorth(g.fpu, p, iq, PosDouble.unsafeFrom(2.0)),
-              if (isFPU) GmosYBinning.One else DefaultYBinning,
```

Comment:
`DefaultBinning` is gone, but it used to be an alias for `GMosYBinning.Two`. I don't know if this is still correct.

Reply:
hmm this is tricky, we may need to replicate the calculation done for each mode:
https://github.com/gemini-hlsw/lucuma-core/pull/810

Reply:
Would that mean calling one of the `*spatialBinning` functions? We would need a `detector` (or `pixelSize`) or default to `Hamamatsu`.

Reply:
we default to `Hamamatsu`

Reply:
I'm not fully aware of the details of the calculation, perhaps @swalker2m could enlighten us

Reply:
Each will provide its own binning calculation implemented to call the more generic `gmos.binning` method as appropriate.  So we wouldn't need to call any `*spatialBinning` directly.  In this case, we'd use `lucuma.core.model.sequence.gmos.longslit.northBinning`.  It defaults to `Hamamatsu` and defaults the sampling to the new preferred value of `2.5`.

So something like `northBinning(g.fpu, p, iq, g.grating)`.

Does that make sense?

Reply:
Yep. And that will give us both the X and Y binning. Thanks!

Reply:
Oh, one other thing: Do I not need the `isFPU` anymore? Or is that still necessary?

Reply:
Thanks. I'll push an update in a couple of minutes.

Reply:
One other thing.  The `isIFU` check is already defined in `GmosNorthFpu` and `GmosSouthFpu` so that definition can be deleted and then you can call `g.isIFU` directly.

Reply:
Cool. Pushed that change, too. 😄